tests/logictest/suites/base/05_ddl/05_0024_ddl_set_role

```diff
@@ -1,12 +1,6 @@
 statement ok
 CREATE ROLE IF NOT EXISTS 'test-s'
 
-statement query T
```

Comment:
The test is useless and flaky...

We can't guarantee the current role is `account_admin`.

Reply:
cc @flaneur2020 Need help, how to test current_role()?

Reply:
15~16 rows test `current_role()`.  The test is useless

Reply:
I think maybe want to test `SELECT current_role()!='test-s';`? I'm not sure.

Reply:
oh, 'account_admin' is the default role of the user which executing sqlogictest in the before. IMHO this test case can be skipped because this checks a default value but not the behaviour of the `set role` statement.